src/test/java/com/dsrm/dsrmbackend/UserControllerIntegrationTests.java

```diff
@@ -30,6 +39,9 @@ class UserControllerIntegrationTests extends  AbstractIntegrationTest{
     @Autowired
     private WebApplicationContext context;
 
+    @Autowired
+    private UserService userService;
```

Comment:
Why?

Reply:
I wanted to test the operation of the method in UserService

Reply:
I would prefer if you use mockmvc instead

Reply:
I would prefer if you use mockmvc instead. In previous test retrieve user by using repo and check if user roles has the same id and name as you set in patch method. And remove this test